adafruit_logging.py

```diff
@@ -143,91 +177,104 @@ def getEffectiveLevel(self):
         """
         return self._level
 
-    def addHandler(self, hldr):
+    def addHandler(self, handler):
         """Sets the handler of this logger to the specified handler.
         *NOTE* this is slightly different from the CPython equivalent which adds
-        the handler rather than replaceing it.
+        the handler rather than replacing it.
 
-        :param hldr: the handler
+        :param handler: the handler
```

Comment:
Since we're patching docstrings - would we also want to document the type of `handler` here?

Reply:
Yes, I didn't understand what the type is supposed to be since the `Logger.addHandler(..)` method is not used anywhere in this file or the examples. But I now see it's supposed to be any subclass of the `LoggingHandler` abstract class.